src/main/asciidoc/index.adoc

```diff
@@ -342,6 +354,18 @@ This handles integration with the https://docs.spring.io/spring-framework/refere
 
 In general, these components make it possible to use standard java exception handling (throw and try-catch) for dealing with problems in Spring Boot REST APIs.
 
+[[belgif-rest-problem-spring-boot-2]]
+=== belgif-rest-problem-spring-boot-2
+
+Rather than depending on `belgif-rest-problem-spring` directly, Spring Boot 2.x users are recommended to depend on `belgif-rest-problem-spring-boot-2`, which adds some Spring Boot 2.x specific integrations.
```

Comment:
Should we change `belgif-rest-problem-spring` to a section reference? Now we have to scroll up to the start when we want to see its actual features.